### DISCUSSION TITLE
Do I need to purchase a domain with SSL for my Vercel app?

### DISCUSSION THREAD

**Discussion by randomprogramming** [vercel/vercel#6313] [Help] [answered]
Hello, I want to buy a domain for my app which is hosted on Vercel. Since I am currently using a .vercel.app subdomain, I have SSL on my website. I am wondering if I have to pay for SSL when purchasing a domain from a site like godaddy or is that not necessary?

**Comment by remjx** [comment, marked as answer]
No you don't need to purchase SSL, Vercel does that automatically when you connect your custom domain

**Comment by fu-sen** [comment]
A little more detailed Vercel official information:
https://vercel.com/blog/automatic-ssl-with-vercel-lets-encrypt

This is an old blog article, so please replace some `.now.sh` with` .vercel.app`. The CLI display is also currently changing.
